imgoing-frontend/src/components/HomeLanding.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { View, Image } from 'react-native';
+import CalloutTypo from './typography/CalloutTypo';
+import styled from 'styled-components/native';
+import FootnoteTypo from './typography/FootnoteTypo';
+import NotificationBar from './NotificationBar';
+import RoundButton from './common/RoundButton';
+
+const ImgView = styled.Image`
+  height: 180px;
+  width: 260px;
+  margin-top: 119px;
+  margin-bottom: 40.99px;
+`;
+
+const CalloutTypoView = styled.View`
+  align-items: center;
+  height: 46px;
+  width: 100%;
+  margin-bottom: 16px;
+`;
+
+const FootnoteTypoView = styled.View`
+  align-items: center;
+  width: 100%;
+  height: 34px;
+  margin-bottom: 108px;
+`;
+
+const ButtonView = styled.View`
+  flex-direction: row;
+  justify-content: center;
+  height: 50px;
+  width: 100%;
+  margin-bottom: 24px;
+`;
+
+const Gap = styled.View`
+  width: 12px;
+`;
+
+const NotificationBarView = styled.View`
+  width: 100%;
+  padding-top: 110px;
+`;
+
+const HomeLanding = () => {
+  const [isNotiBarVisible, setIsNotiBarVisible] = useState(true);
```

Comment:
```suggestion
  const [isNotiBarVisible, setIsNotiBarVisible] = useState<boolean>(true);
```

타입 명시해주면 더 좋을 것 같아요😊